### PR TITLE
Add Sitting Framework Documentation

### PR DESCRIPTION
**Pull request by jonpas**
This will be live in 3.4.0, currently in master.

Tagging @alganthe

### CODE REVIEW

File: wiki/framework/sitting-framework.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: Sitting Framework
+description: Explains how to set-up sitting objects (eg. chairs) with ACE3 sitting system.
+group: framework
+order: 5
+parent: wiki
+---
+
+## 1. Requirements
+
+Object must inherit from `ThingX` or any of its sub-classes.
```
Comment (alganthe) [inline wiki/framework/sitting-framework.md]: `or any of its sub-classes.`
to 
`or any of *it's* sub-classes.`

Comment (jonpas) [inline wiki/framework/sitting-framework.md]: It's meant as "its" and not "it is". What you wrote would mean "or any of it is sub-classes", no?

Comment (alganthe) [inline wiki/framework/sitting-framework.md]: Actually, yep.

Everything seems good :+1:

Comment (jonpas) [inline wiki/framework/sitting-framework.md]: Good thing neither of us are natives to English! :grin:

**Comment by PabstMirror** [comment]
This should probably wait until after 3.4's release if it isn't in current release.

**Comment by jonpas** [comment]
Yes, when 3.4.0 releases.

**Comment by jonpas** [comment]
This is ready for merge.

**Pull request merged**